packages/plugin-multi-tenant/src/fields/tenantsArrayField/index.ts

```diff
@@ -4,17 +4,19 @@ export const tenantsArrayField = (args: {
   arrayFieldAccess?: ArrayField['access']
   rowFields?: ArrayField['fields']
   tenantFieldAccess?: RelationshipField['access']
+  tenantFieldName: string
+  tenantsCollectionSlug: string
 }): ArrayField => ({
-  name: 'tenants',
+  name: args.tenantsCollectionSlug,
```

Comment:
To really support this we should have a specific property for the `tenantsArrayFieldName`